pkgs/development/libraries/qwt/6_qt4.nix

```diff
@@ -18,6 +18,13 @@ stdenv.mkDerivation rec {
     sed -e "s|QWT_INSTALL_PREFIX.*=.*|QWT_INSTALL_PREFIX = $out|g" -i qwtconfig.pri
   '';
 
+  # qwt.framework output includes a relative reference to itself, which breaks dependents
+  fixupPhase =
```

Comment:
I think this should use preFixup (or postFixup), unless you really mean to override the fixupPhase. (In that case, a comment is required.)

Reply:
👍